doc/047_tuning_backup_parameters.rst

```diff
@@ -47,3 +48,26 @@ is compressed with the option ``--compression``. It can be set to ```auto``` (th
 which will compress very fast), ``max`` (which will trade backup speed and CPU usage for
 slightly better compression), or ``off`` (which disables compression). Each setting is
 only applied for the single run of restic.
+
+
+Pack Size
+=========
+
+In certain instances, such as very large repositories (in the TiB range) or very fast
+upload connections, it is desirable to use larger pack sizes to reduce the number of
+files in the repository and improve upload performance.  Notable examples are OpenStack
+Swift and some Google Drive Team accounts, where there are hard limits on the total
+number of files.  Larger pack size can also improve the backup speed for a repository
```

Comment:
"Larger pack size" => "Larger pack sizes"?

Reply:
Is fixed.